src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,108 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+import { getMovie } from '../../api/api';
+
+export const FindMovie = ({ addMovie }) => {
+  const [movie, setMovie] = useState({});
```

Comment:
```suggestion
  const [movie, setMovie] = useState(null);
```
Than you can just check `if (movie)` instead of using `Object.keys` in `handleSubmit`